scripts/prerelease/generate_prerelease_script.py

```diff
@@ -77,6 +78,12 @@ def main(argv=sys.argv[1:]):
     build_files = get_source_build_files(config, args.rosdistro_name)
     build_file = build_files[args.source_build_name]
 
+    # Use targets defined by corresponding Release config instead of
+    # targets defined by Source config. This is to increase the number of
+    # supported targets. New targets are explicitly passed to configure_devel_job().
+    release_build_files = get_release_build_files(config, args.rosdistro_name)
+    release_build_file = release_build_files[args.source_build_name]
```

Comment:
This won't work well since the `source_build_name` might not be the same. E.g. their is an `arm` release build file with no equivalent source build file.